### ISSUE TITLE
PeerConnection crash Unity when AutoInitializeOnStart is true

### ISSUE THREAD

**Issue by amfdeluca** [microsoft/MixedReality-WebRTC#218]
Hi, everytime i check AutoInitializeOnStart on PeerConnection my Unity crashes, i have already used webRTC in another project, times ago, and all worked well, I have in scene The CustomSignaler component and CustomSignalerUI.  Tell me if you need more information or if it is a common problem and i'm only missing something obvious.

**Comment by djee-ms** [comment]
Hi @amfdeluca, no never heard of that, we will need more info. For both peers: are you running inside the Unity editor or as a standalone build? What audio/video codecs are you using, if any. What branch/commit are you on? Do you have a microphone and webcam plugged in the device, and not disabled?

**Comment by amfdeluca** [comment]
Hi, thanks for the quick reply, I tried to re-import MixedReality-WebRTC 1.0.3 as I did not remember the previous version and therefore I followed the installation step by step:
- Added the two folder while unity was closed: Microsoft.MixedReality.WebRTC.Unity and Microsoft.MixedReality.WebRTC.Unity.Editor
- I then added the Plugins folder with the meta files inside which I then associated the corresponding .dll by following this guide (https://microsoft.github.io/MixedReality-WebRTC/manual/installation.html#copying-the-native -dlls) version 1.0.2 from the NuGet packages website.
Checking the summary is therefore all correct (https://microsoft.github.io/MixedReality-WebRTC/manual/installation.html#summary).

After this I then added the two scripts PeerConnection and CustomSignaler to the scene and assigned the corresponding references in the editor.

I then installed and started the Node-dss server that I had previously used.

The problem persists, the editor crashes immediately. Unity 2018.4.9f.
No error and crash if i click play and I uncheck AutoInitializeOnStart from PeerConnection.

Thanks for your attention!
